config/parser.go

```diff
@@ -77,6 +77,8 @@ func (p *Parser) parseLines(lines []string) (err error) {
 			p.opts.logDateTime = parseBool(value, defaultLogDateTime)
 		case "DEBUG":
 			p.opts.debug = parseBool(value, defaultDebug)
+		case "SERVER_TIMING_HEADER":
```

Comment:
I can see you have updated the variable here, but all occurrences should be updated as well.

Reply:
Sorry, it should be fixed now.
Unless you meant to change the timing variable as well?